Gulpfile.ts

```diff
@@ -748,7 +745,7 @@ gulp.task("runtests",
 const nodeServerOutFile = "tests/webTestServer.js";
 const nodeServerInFile = "tests/webTestServer.ts";
 gulp.task(nodeServerOutFile, false, [servicesFile], () => {
-    const settings: tsc.Settings = getCompilerSettings({}, /*useBuiltCompiler*/ true);
+    const settings: tsc.Settings = getCompilerSettings({module: "commonjs"}, /*useBuiltCompiler*/ true);

```

Comment:
Additionally, is this supposed to be consistent with binding patterns and/or template strings? I'm internally consistent in the Gulpefile (no leading/trailing whitespace for any), but again, we're inconsistent within the compiler.